src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -100,6 +102,8 @@ public override IReadOnlyCollection<IHttpCookie> Cookies
 
         public override Uri Url => _url ??= new Uri(_httpData.Url);
 
+        public override IDictionary<string, StringValues> Query => _query ??= _httpData.NullableQuery.ToDictionary(q => q.Key, q => new StringValues(q.Value?.Value.ToString().Split(',')));
```

Comment:
Do we have a test that tests out this logic?

Reply:
That is a good point - not one that tests multiple values for one key. I will add one.